src/sentry/release_health/metrics_sessions_v2.py

```diff
@@ -258,7 +257,7 @@ def get_values(
             started = int(data_points[key])
             abnormal = int(data_points.get(replace(key, raw_session_status="abnormal"), 0))
             crashed = int(data_points.get(replace(key, raw_session_status="crashed"), 0))
-            errored_key = replace(key, metric_name="session.error", raw_session_status=None)
+            errored_key = replace(key, metric_name=MetricKey.SESSION_ERROR, raw_session_status=None)
```

Comment:
Should this be a `.value` ?

Reply:
I changed the type of `metric_name` to `MetricKey`, but did not to rename the variable itself. Renamed to `metric_key` now to make it more clear.